cluster/swarm/cluster.go

```diff
@@ -295,3 +295,15 @@ func (c *Cluster) Info() [][2]string {
 
 	return info
 }
+
+// RANDOMENGINE returns a random engine.
+func (c *Cluster) RANDOMENGINE() (*cluster.Engine, error) {
```

Comment:
Nit-picking on the name but why keeping **random*** if it defaults to the strategy used? (can be something like *PickEngine* or else?)